pkg/nsx/services/securitypolicy/builder.go

```diff
@@ -957,13 +957,6 @@ func (service *SecurityPolicyService) buildPeerTags(obj *v1alpha1.SecurityPolicy
 		groupTypeTag = String(common.TagValueGroupSource)
 		peers = &rule.Sources
 	}
-
-	// TODO: abstract sort func for both peers and targets
```

Comment:
It has the same reason why re removed this sort function for target tags.
This sort function will impact rule hash when generating rule hash for peer groups.